Code/Engine/RendererFoundation/Device/Implementation/Device.cpp

```diff
@@ -642,6 +642,46 @@ ezGALTextureHandle ezGALDevice::CreateTexture(
   return ezGALTextureHandle();
 }
 
+ezResult ezGALDevice::ReplaceExisitingNativeObject(ezGALTextureHandle hTexture, void* pExisitingNativeObject)
+{
+  ezGALTexture* pTexture = nullptr;
+  if (m_Textures.TryGetValue(hTexture, pTexture))
+  {
+    for (auto it = pTexture->m_RenderTargetViews.GetIterator(); it.IsValid(); ++it)
+    {
+      ezGALRenderTargetView* pRenderTargetView = nullptr;
+
+      if (m_RenderTargetViews.TryGetValue(it.Value(), pRenderTargetView))
+      {
+        EZ_VERIFY(pRenderTargetView->DeInitPlatform(this).Succeeded(), "DeInitPlatform should never fail.");
+      }
+    }
+
+    EZ_VERIFY(pTexture->DeInitPlatform(this).Succeeded(), "DeInitPlatform should never fail.");
+    EZ_VERIFY(pTexture->ReplaceExisitingNativeObject(pExisitingNativeObject).Succeeded(),
+      "Failed to replace native texture, make sure the input is valid.");
+    EZ_VERIFY(pTexture->InitPlatform(this, {}).Succeeded(),
+      "InitPlatform failed on a texture the previously succeded in the same call, is the new native object valid?");
+    
+    for (auto it = pTexture->m_RenderTargetViews.GetIterator(); it.IsValid(); ++it)
+    {
+      ezGALRenderTargetView* pRenderTargetView = nullptr;
+
+      if (m_RenderTargetViews.TryGetValue(it.Value(), pRenderTargetView))
+      {
+        EZ_VERIFY(pRenderTargetView->InitPlatform(this).Succeeded(),
+          "InitPlatform failed on a texture the previously succeded in the same call, is the new native object valid?");
+      }
+    }
+    return EZ_SUCCESS;
+  }
+  else
+  {
+    ezLog::Warning("DestroyTexture called on invalid handle (double free?)");
```

Comment:
>DestroyTexture [](start = 20, length = 14)

should be ReplaceExisitingNativeObject and remove the (double free?) at the end

Reply:
Copy paste error :)